runtime-local/test/integration/integration_test.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2023 Robert Bosch GmbH
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+from pathlib import Path
+from re import compile, Pattern
+from subprocess import PIPE, Popen
+from threading import Timer
+
+
+def create_dummy_vspec_file():
+    """Creates an empty vspec json file in the project's cache directory
+    plus the respective 'vspec_file_path' cache variable pointing to it.
+    !!! Todo: Replace this workaround solution making assumptions about
+              the cache location.
+    """
+    cache_paths = Path("~/.velocitas/projects").expanduser().rglob("cache.json")
+    for cache_path in cache_paths:
+        vspec_path = cache_path.parent / "dummy_vspec.json"
```

Comment:
use join rather than string concat

Reply:
This is the way!
See https://docs.python.org/3/library/pathlib.html#operators

Reply:
Interesting, thanks for pointing that out.